resources/views/home.blade.php

```diff
@@ -74,7 +74,6 @@
                         <div class="form-group">
                             {{ Form::label('copy_from', trans('dashboard.label.copy')) }}
                             <select class="form-control" id="copy_from" name="copy_from">
-                                <option></option>
```

Comment:
You don't necessarily want to copy from another dashboard so you need the blank option.